### PR TITLE
Changed minor wordings.

### PR DESCRIPTION
**Pull request by initbar**
There were couple of wrong highlights due to `'` and `''` - so they are now escaped.

### CODE REVIEW

**Comment by YASME-Tim** [comment]
Unfortunately there is a trade-off between correct highlighting (in vim e.g.) and the correctness of the resulting text. E.g. look at the command: If a beginner copies that, the command won't work. I don't have a solution for the highlighting problem yet, so I mostly now leave it as is or only escape things that are not affected by an additional '\'. Thanks though @initbar

File: opentoall-ctf-2015/forensics/lost_flug/README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ arB=NICK logman1337
 ardC)oR1:logman1337!~logman@162.245.177.238 JOIN #XPvCxHRTCFKBBzyb
 :rajaniemi.freenode.net 353 logman1337 @ #XPvCxHRTCFKBBzyb :logman1337 @kee7a
 :rajaniemi.freenode.net 366 logman1337 #XPvCxHRTCFKBBzyb :End of /NAMES list.
-)oS":rajaniemi.freenode.net 352 logman1337 #XPvCxHRTCFKBBzyb ~logman 162.245.177.238 rajaniemi.freenode.net logman1337 H :0 logman
+)oS\":rajaniemi.freenode.net 352 logman1337 #XPvCxHRTCFKBBzyb ~logman 162.245.177.238 rajaniemi.freenode.net logman1337 H :0 logman
```
Comment (mathiasbynens) [inline opentoall-ctf-2015/forensics/lost_flug/README.md]: ??

File: opentoall-ctf-2015/misc/secure-server/README.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ When running `nmap -p0-65535 -T5 104.131.124.226`, wee see that there are two pu
 $ nmap -p0-65535 -T5 104.131.124.226
 Starting Nmap 6.47 ( http://nmap.org ) at 2015-03-06 14:12 CET
 Warning: 104.131.124.226 giving up on port because retransmission cap hit (2).
-Strange error from connect (49):Can't assign requested address
+Strange error from connect (49):Can\'t assign requested address
```
Comment (mathiasbynens) [inline opentoall-ctf-2015/misc/secure-server/README.md]: ??

File: opentoall-ctf-2015/misc/secure-server/README.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ We see a `robots.txt` file that gives us another directory to search for any hid
 Inside `http://104.131.124.226:8080/secret`, we find a [secret.zip](./104.131.124.226\:8080/secret/secret.zip) file that we download and extract:
 
 ```bash
-$ unzip secret.zip && tree -a
+$ unzip secret.zip \&\& tree -a
```
Comment (mathiasbynens) [inline opentoall-ctf-2015/misc/secure-server/README.md]: ??

Comment (YASME-Tim) [inline opentoall-ctf-2015/misc/secure-server/README.md]: I did revert that one. @initbar should mention the editor used, though, which breaks on a `'`, `&` or `"`. Failed to verify before, thought it is a similar problem as `_` in vim for another editor.

File: opentoall-ctf-2015/misc/secure-server/README.md (hunk 2 above)
Comment (dhanvi) [inline opentoall-ctf-2015/misc/secure-server/README.md]: that's a conflict with the mark down format highlighting  @mathiasbynens

Comment (YASME-Tim) [inline opentoall-ctf-2015/misc/secure-server/README.md]: not with vim though. vim only breaks with `_` as far as I know

**Pull request merged**